rpm/pacemaker.spec.in

```diff
@@ -274,6 +274,7 @@ Requires:      resource-agents
 Requires:      %{pkgname_pcmk_libs}%{?_isa} = %{version}-%{release}
 Requires:      %{name}-cluster-libs%{?_isa} = %{version}-%{release}
 Requires:      %{name}-cli = %{version}-%{release}
+Requires:      %{python_name}-%{name} = %{version}-%{release}
```

Comment:
I just noticed we already have `%bcond_with stonithd` to enable the support that fence_legacy is needed for, so we can guard this and whether to install fence_legacy